exec/java-exec/src/main/java/org/apache/drill/exec/cache/VectorAccessibleSerializable.java

```diff
@@ -127,6 +129,60 @@ public void readFromStream(InputStream input) throws IOException {
     va = container;
   }
 
+  // Like above, only preserve the original container and list of value-vectors
```

Comment:
This class was refactored in the external sort unit test PR. Let's coordinate on merging the two sets of changes.